docker/run_jakartaeetck.sh

```diff
@@ -455,7 +455,7 @@ else
 fi
 
 ENABLE_RERUN="${ENABLE_RERUN=true}"
-if [ "$ENABLE_RERUN" -eq true ]; then
+if [ "$ENABLE_RERUN" -eq "true" ]; then
```

Comment:
```
       STRING1 = STRING2
              the strings are equal

       STRING1 != STRING2
              the strings are not equal

       INTEGER1 -eq INTEGER2
              INTEGER1 is equal to INTEGER2
```


```
$ bash -c 'if [ "true" -eq "true" ]; then echo 1; fi'
bash: line 1: [: true: integer expression expected
```

Reply:
I had this exact error before I added quotation marks. Now I am pretty confused, but thank you for quick review. I feel pretty stupid now and I am very sorry I did not wait a bit.

Reply:
Oh, I see, double square brackets!